lib/Parser/Parser.cpp

```diff
@@ -386,6 +387,14 @@ Inst *Parser::parseInst(std::string &ErrStr) {
   }
 }
 
+bool Parser::isOverflow(Inst::Kind IK) {
+  if (IK == Inst::SAddWithOverflow || IK == Inst::UAddWithOverflow ||
```

Comment:
you don't need an "if" in this kind of function, just return the result of the or-expression